source/user_guide/basic_capabilities/feature_attributes.rst

```diff
@@ -59,11 +59,40 @@ What is the difference between **nominal**, **ordinal**, and **continuous** feat
 How do I map **ordinal** features?
 ----------------------------------
 - If the feature is `numeric`, all you must do is specify the ``type`` as
-  `ordinal` inside IFA.
+  `ordinal` in the feature attributes dictionary.
 - If the feature is `ordinal` but not `numeric`, pass a dictionary specifying
   the order to IFA using the ``ordinal_feature_values`` argument. - An example
   is: ``{ "size" : [ "small", "medium", "large", "huge" ] }``
 
+How can I preset **nominal** and **continuous** feature types?
+------------------------------
+- Use the ``types`` parameter to ensure that ``infer_feature_attributes`` treats certain features
+  as either `nominal` or `continuous`.
+
+- This is important because IFA may not correctly specify the bounds if the feature type is
+  ambiguous. For example, imagine a `nominal` feature of US zip codes (90016,
+  91334, etc.). IFA may infer these values to be `continuous` and the resulting
+  dictionary will include ``min`` and ``max`` bounds. You can edit the `type` to be
+  `nominal` post calling IFA, but the `continuous` bounds may cause an issue
+  when reacting to the model. This is why it's often better to use ``types``.
+
+  .. code-block:: python
+
+    # Ensure that potentially ambiguous types are preset.
```

Comment:
What happens if I have a feature named `ordinal`? Seems risky to allow both directions. I'd lean towards _just_ the type list version.

Reply:
This would make a good unit test, tbh.